### PR TITLE
fix: Reimplement the "unknown owner" option to plot purge

### LINKED ISSUE #3353: plot purge with optional unknown: argument doesn't work
### Server Implementation

Paper

### Server Version

1.17.1

### Describe the bug

When using `/plot purge world:plotsquared unknown:true clear:true` the command doesn't work and just prints the command usage again

Untitled

By removing the optional unknown argument the purge command works. I'm not sure if the `unknown:` argument is no longer supported or if it just doesn't work but if it's no longer supported then removing it from the command usage would be helpful

### To Reproduce

1. Use `/plot purge world:[worldname] unknown:true clear:true`

### Expected behaviour

Command usage shouldn't show `unknown:` as a valid optional argument if it is no longer supported

### Screenshots / Videos

_No response_

### Error log (if applicable)

_No response_

### Plot Debugpaste

https://athion.net/ISPaster/paste/view/a3f2db337da341f2968b6f6f00e84f97

### PlotSquared Version

PlotSquared-Bukkit-6.1.4-Premium

### Checklist

- [X] I have included a Plot debugpaste.
- [X] I am using the newest build from https://www.spigotmc.org/resources/77506/ and the issue still persists.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by dordsor21**
- Warn user that backgorund UUID caching may fix the issue
- Also correct the UUID set when purging based on "shared" (added) player
- Fixes #3353

### CODE REVIEW

File: Core/src/main/resources/lang/messages_en.json [hunk 1]
```diff
@@ -322,6 +322,8 @@
   "debugsavetest.starting": "<prefix><gold>Starting debugsavetest.</gold>",
   "debugsavetest.done": "<prefix><gold>Database sync finished.</gold>",
   "purge.purge_success": "<prefix><dark_aqua>Successfully purged <amount> plots.</dark_aqua>",
+  "purge.confirm_purge_unknown_bg_enabled": "<prefix><gray>Background UUID caching is enabled. Unknown plot owners may be found using this!</gray>",
+  "purge.confirm_purge_unknown_bg_disabled": "<prefix><gray>Background UUID caching is disabled. Unknown plot owners may be found using this! Enabled in settings.yml</gray>",
```
Comment (Citymonstret) [inline Core/src/main/resources/lang/messages_en.json:326]: Should this say `Enable` instead of `Enabled`? Sounds quite confusing otherwise

Comment (dordsor21) [reply Core/src/main/resources/lang/messages_en.json]: oops

**Review by NotMyFault — APPROVED** [review]

**Review by yannicklamprecht — APPROVED** [review]

**Review by SirYwell — APPROVED** [review]

**Comment by NotMyFault** [comment]
Quick fix, we may merge it in ~24 hours if there is no negative feedback, as per the [merge process](https://github.com/IntellectualSites/IntellectualSites/blob/main/docs/MAINTAINERS_CONTRIBUTORS.adoc#merge-process) documentation.

**Pull request merged**